README.md

```diff
@@ -1,40 +1,79 @@
 # Signon
 
-Signon is a centralised OAuth2 based single sign-on provider for GDS services that provides username/password and 2-Factor authentication.
+Signon is a centralised, OAuth2-based single sign-on (SSO) provider for GDS services that provides username/password and 2-factor authentication.
 
-## Technical documentation
+Signon uses [Devise] for username/password sign-in, and [Doorkeeper] as an OAuth 2 provider. Details of our interpretation of OAuth are provided in [an accompanying document](docs/oauth.md).
 
-[Devise] is used to provide username password sign-in, and [Doorkeeper] as an OAuth 2 provider. Details of our interpretation of OAuth are provided in [an accompanying document][auth].
+Signon is a Ruby on Rails app and should follow [our Rails app conventions][conventions].
 
-This is a Ruby on Rails app, and should follow [our Rails app conventions][conventions].
+[Devise]: https://github.com/heartcombo/devise
+[Doorkeeper]: https://github.com/doorkeeper-gem/doorkeeper
+[conventions]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+
+
+## Running the test suite using GOV.UK Docker
+
+You can use the [GOV.UK Docker environment][govuk-docker] to run the application, its tests and all its dependencies. Follow the [govuk-docker installation instructions][govuk-docker-install] to get started.
 
-You can use the [GOV.UK Docker environment][govuk-docker] to run the application and its tests with all the necessary dependencies. Follow the [usage instructions][docker-usage] to get started.
+The first build with GOV.UK Docker typically takes 5-20 minutes, depending on your computer and network connection.
```

Comment:
This information seems like it should be in the GOV.UK docker documentation rather than in here